### DISCUSSION TITLE
Nvidia shield getting "Invalid status code from API: 404"

### DISCUSSION THREAD

**Discussion by Daniel-PT** [giejay/Immich-Android-TV#29] [Q&A]
Hi all
Tryed this app some time ago and it is wokring fine.
But today when i tryed it i got 
"Invalid status code from API: 404"
I tryed
A new API key
Reinstall the app
restart Nvidia Shield.
What can i do?
![image](https://github.com/giejay/Immich-Android-TV/assets/20338866/47e9efb7-1846-4ced-b3d9-8f1c8c97b5e0)

Thanks.

**Comment by giejay** [comment]
Can you turn on debug logging and send me your user id?

**Reply by Daniel-PT** [reply]
Off course :)
How can i sent the user id "private"? :)

**Reply by giejay** [reply]
It's not sensitive, it's simply generated by the app on bootup so you can just paste it here

**Reply by Daniel-PT** [reply]
okay :)
a2e7a859-9de1-44d1-896c-693d7c900ef3

**Reply by giejay** [reply]
Can't find anything under that user id. Are you using system wide adblockers?

**Reply by Daniel-PT** [reply]
Sorrt @giejay i had one digit wroung :)

**Reply by giejay** [reply]
Your server is indeed giving back 404. Can you try going to this url manually: http://192.168.1.50:8089/api/assets?page=1&size=100&order=desc

**Reply by Daniel-PT** [reply]
Is there any easy way to insert that? :) Hard to type ;)

**Reply by Daniel-PT** [reply]
After i did that the app crashes everytime i open it :(

**Reply by giejay** [reply]
I think the assets endpoint is removed in a new version... Not sure if there's a good alternative

**Reply by Daniel-PT** [reply]
New ID:
73de836a-e386-46a9-99ec-c94b7dbbc281
Had to reinstall the app.

**Reply by giejay** [reply]
No, I meant simply going to this url from your phone/laptop

**Reply by Daniel-PT** [reply]
Ahh sorry :)
I get:
{"message":"Cannot GET /api/assets?page=1&size=100&order=desc","error":"Not Found","statusCode":404}

**Reply by bo0tzz** [reply]
You want the new /api/asset endpoint instead :) https://immich.app/docs/api/get-all-assets

**Reply by giejay** [reply]
That endpoint does not allow sorting of the output. I moved to search/metadata instead in the latest release and that seems to be the replacement. Please try the latest release and the 404 should be gone.

**Reply by Daniel-PT** [reply]
Hi :)
Thanks for the quick work! :)
But i cant see update it?
My Nvidia shield dosent see an update?
Is the latest version 1.32?
https://play.google.com/store/apps/details?id=nl.giejay.android.tv.immich
Thanks!

**Reply by davralin** [reply]
@giejay - google play still only has 1.34, and the release-page doesn't have the APK-asset attached to 1.35 - is there any other ways to get the latest apk from you? Not too familiar with android-stuff, but I believe that if I build it myself, there are other keys in use, which would stop me from updating later on..

**Reply by giejay** [reply]
Thought I uploaded it but just saw that the upload failed. Have added it now

**Reply by davralin** [reply]
I'm still getting 404's from that release, and the version tag is 1.36.1

**Comment by Daniel-PT** [comment]
Nope.

I think i have notet the correct id? 🙂



tors. 2. maj 2024 21.10 skrev GJ Compagner ***@***.***>:

**Comment by giejay** [comment]
Yeah this should be the correct id

**Comment by giejay** [comment]
It will take some time. Give it 2 hours

**Reply by Daniel-PT** [reply]
Hi again :)
Strange it has not updated. I know that google play can be slow but is version 1.32 still det newest? :)
Thanks!

**Reply by giejay** [reply]
It's still being reviewed, I see. You can try sideloading it, or wait a bit more

**Reply by Daniel-PT** [reply]
Hi :)

I has now updated. But the error is still present :(

Can i do something?

**Reply by giejay** [reply]
Are you using the latest version of Immich server?

**Reply by Daniel-PT** [reply]
Im running v1.103.1 so yeap latest :)

**Comment by giejay** [comment]
What is your current hostName?

**Reply by Daniel-PT** [reply]
Ahh that did the trick :) I removed /api :)

Thanks for the help!

**Comment by giejay** [comment]
Does the demo mode work for you?

**Reply by davralin** [reply]
If you were supposed to ask me, the answer is yes.
Guess that means something is wrong on my end then?

**Reply by giejay** [reply]
Correct. Check if you are using the latest immich server
